volumina/widgets/slotMetaInfoDisplayWidget.py

```diff
@@ -45,18 +45,27 @@ def initSlot(self, slot):
         self._refresh()
 
     def _refresh(self, *args):
+        shape = axes = dtype = ""
         if self._slot.ready():
             shape = tuple(self._slot.meta.getOriginalShape())
```

Comment:
no biggy, really and I know there are no types in python, but since you initialized shape as a string, i would expect it to stay one?

Reply:
I agree, switching types is for savages D=
I should stop rushing these bug fixes